Sources/ProjectSpec/Scheme.swift

```diff
@@ -364,6 +367,10 @@ extension Scheme.Run: JSONObjectConvertible {
         } else if let string: String = jsonDictionary.json(atKeyPath: "launchAutomaticallySubstyle") {
             launchAutomaticallySubstyle = string
         }
+        
+        if let askLaunch: Bool = jsonDictionary.json(atKeyPath: "askForAppToLaunch") {
```

Comment:
How about defaulting to `false`?

Reply:
the key is by default non-existing instead of false

Reply:
If this value were false, then the property would be nil. Is that what is intended?